crates/wdk-build/rust-driver-makefile.toml

```diff
@@ -391,64 +523,48 @@ run_task = "signtool-verify"
 [tasks.package-driver]
 private = true
 dependencies = [
-  "copy-sys-to-package",
+  "copy-driver-binary-to-package",
   "copy-pdb-to-package",
   "copy-inf-to-package",
   "copy-map-to-package",
   "copy-certificate-to-package",
-  "sign-sys",
-  "verify-signature-sys",
+  "sign-driver-binary",
+  "verify-signature-driver-binary",
   "sign-cat",
   "verify-signature-cat",
   "infverif",
 ]
 
 [tasks.package-driver-flow]
-# Only run flow if the current package is marked as a driver
+# Note: Dependencies are always run, regardless of the condition_script result. This allows `cargo make` in mixed driver/non-driver workspaces
+dependencies = ["build"]
+# Only run package-driver flow if the current package is marked as a driver
+plugin = "nested-cargo-workspace-in-cargo-make-emulated-workspace-support"
+condition_script_runner_args = [
+  "--base-path",
+  "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}",
```

Comment:
The proper expansion of this is blocked on https://github.com/sagiegurari/cargo-make/pull/1132

Reply:
added workaround to unblock pr